### DISCUSSION TITLE
Cantaloupe problems!

### DISCUSSION THREAD

**Discussion by Marcolamorte** [cantaloupe-project/cantaloupe#679] [General]
Subject: Assistance Required with Implementing Cantaloupe IIIF Image Server
Dear Cantaloupe Team,
I hope this message finds you well. I am reaching out to seek assistance with the implementation of the Cantaloupe IIIF Image Server, which I have been struggling with over the past few days. Despite following the documentation closely, I am encountering persistent issues that I have been unable to resolve on my own.
I have
- Operating System: Windows 10
- Java Version: 20 (build 20+36-2344)
- Cantaloupe Version:  5.0.6
- Image Directory Path: `C:\Users\User\Desktop\unibo\Mancinelli\Images_IIIf\Madorski`
- Configuration File Path: `C:\Users\User\Desktop\unibo\Mancinelli\cantaloupe-5.0.6\cantaloupe.properties`

1. I have set the `FilesystemSource.BasicLookupStrategy.path_prefix` in the `cantaloupe.properties` file to the correct image directory:
   ```FilesystemSource.BasicLookupStrategy.path_prefix = C:/Users/User/Desktop/università_bolo/Mancinelli/Images_IIIf/Madorski/ ```
2. I attempted to start Cantaloupe using the following command:
   ```cmd java -Dcantaloupe.config=C:\Users\User\Desktop\università_bolo\Mancinelli\cantaloupe-5.0.6\cantaloupe.properties -Xmx2g -jar C:\Users\User\Desktop\università_bolo\Mancinelli\cantaloupe-5.0.6\cantaloupe-5.0.6.jar ```

- When executing the above command, the process starts but then hangs indefinitely without completing. Despite running the Command Prompt as an administrator, I cannot get past this issue.
Given these challenges, I would greatly appreciate your guidance on the matter.
Thank you in advance for your assistance. I am eager to get this IIIF server up and running, and your support would be invaluable in helping me overcome these technical obstacles.

Looking forward to your response.

Best regards,  
Marco Lamorte
Marco.lamorte@studio.unibo.it
Università di Bologna

**Comment by jcoyne** [comment]
@Marcolamorte when you say "hangs indefinitely", is it responding to HTTP requests at this time?  Are you seeing anything in the log files?

**Comment by glenrobson** [comment]
Marco reached out through the IIIF website and let me know he has resolved this issue which was related to the paths.
